### PR TITLE
Upgraded to elasticsearch 5.5.0

### PR DESCRIPTION
**Pull request by marcofranssen**
Upgraded to elasticsearch 5.5.0

### CODE REVIEW

**Comment by jimmidyson** [comment]
ok to test

File: pom.xml [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>io.fabric8</groupId>
   <artifactId>elasticsearch-cloud-kubernetes</artifactId>
-  <version>5.4.3-SNAPSHOT</version>
+  <version>5.5.5-SNAPSHOT</version>
```
Comment (jimmidyson) [inline pom.xml]: This should be 5.5.0-SNAPSHOT please.

**Comment by marcofranssen** [comment]
I also added an improvement on the readme.

**Comment by jimmidyson** [comment]
Awesome thanks! [merge]

**Comment by marcofranssen** [comment]
When can it be expected to be available by elasticsearch-plugin install command?

**Pull request merged**